packages/spacecat-shared-aws-costusage-client/test/index.test.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-env mocha */
+import chai from 'chai';
+
+
+// eslint-disable-next-line import/no-named-default
+import { AWSCostApiClient } from '../src/index.js';
+import chaiAsPromised from 'chai-as-promised';
+import nock from 'nock';
+
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+describe('run aws api client', () => {
```

Comment:
this test does not verify the actual result. We can mock the amazon api behind the service using `nock`. Example for that: https://github.com/adobe/spacecat-api-service/blob/main/test/support/sqs.test.js#L82